src/shared/views/ChangeDetection/Body.js

```diff
@@ -304,7 +306,10 @@ class _ChangeDetectionBody extends React.Component<Props, State> {
                 <View>
                     <TouchableWithoutFeedback
                         onPress={() => {
-                            navigation.push('CDInstructionsScreen');
+                            navigation.push('CDInstructionsScreen', {
```

Comment:
Can this be replaced with `onInfoPress` as the code looks exactly the same?

Reply:
good catch.